src/chapters/git/exercises.rst

```diff
@@ -8,19 +8,117 @@ We will use our new terminal powers to move through the Git exercises.
 
 #. In whichever directory you are keeping your coursework, make a new directory called ``Git-Exercises`` using the ``mkdir`` command. 
 #. Inside the ``Git-Exercises`` directory, initialize a new Git repository using ``git init``.
+#. Use ``git branch`` to check the default branch name. If necessary, change the
+   branch name to ``main``.
 #. Add a file called ``exercises.txt`` using the ``touch`` command in the terminal.
 #. Commit your local changes using the ``git commit`` procedures.
-#. Add "Hello World!" to the file called ``exercises.txt``.
-#. Commit your local changes following the same steps that you used for step 4.
-#. View and screenshot the result when you use ``git log``. Make note of what you see!
+#. Add ``"Hello World!"`` to the file called ``exercises.txt``.
+#. Commit your local changes following the same steps that you used for step 5.
+#. Run the ``git log`` command. Take a screenshot of the result. Make note of
+   what you see!
 
 Setting up a Github Account
 ---------------------------
 
-For our remote repositories, we will be using `GitHub <https://github.com/>`_. 
+For our remote repositories, we will be using `GitHub <https://github.com/>`__. 
 
 To create your account, follow these steps:
 
 #. Navigate to GitHub's site using the link above.
-#. Sign up for an account on the homepage either by filling out the form or clicking the "Sign Up" button.
+#. Click the *Sign Up* button and follow the on-screen directions.
 #. Once you have an account, you are ready to store your remote work.
+
+Before August 13, 2021, when users pushed changes to a remote repository (or
+pulled new content from it), they were prompted to enter their GitHub username
+and password. However, a username/password combination is not the most
+secure method available. This is especially true for people who reuse the same
+credentials across multiple websites. (You know who you are. Stop doing that!)
+
+After 8/13/2021, GitHub changed its policy in an effort to boost account
```

Comment:
totally nit picky, but you have two different date formats a few lines apart from each other.  

(and the use of past tense in the "After..." sentence is because the next class to use this section will be doing so after august 13th?)

Reply:
Got it.  No edits are too small or too picky!  ;-)

I rephrased the second paragraph to clean up the date format and tense.